userprofile/views.py

```diff
@@ -128,5 +128,44 @@ def get_common_objects(self):
         self.privacy_text = privacy_text
         self.note("privacy_text")
 
+class AccessibilityStatementView(UserProfileView):
+    access_mode=ACCESS.ANONYMOUS
+    template_name="userprofile/accessibility.html"
+
+    def get_common_objects(self):
+        super().get_common_objects()
+        lang = "_" + get_language().lower()
+        key = make_template_fragment_key('accessibility_statement', [lang])
+        accessibility_statement = cache.get(key)
+        if not accessibility_statement:
+            local_template_name = "institution_accessibility_text{}.html"
+            local_template = try_get_template(local_template_name.format(lang))
+            if not local_template and len(lang) > 3:
```

Comment:
This logic should be refactored to a function for all of the 3 uses (1 in privacy notice and 2 times here), but I think it can be left for later. Can you add a comment `# TODO: refactor to a helper function`?